draft-normington-jsonpath-latest.md

```diff
@@ -425,22 +428,25 @@ An array index is a union element consisting of an integer (in base 10).
 ~~~~ abnf
 array-index = integer
 
-integer = [%x2D] (%x30 / (%x31-39 *%x30-39))
+integer = ["-"] ("0" / (%x31-39 *%x30-39))
```

Comment:
What was the guidance around using quoted characters vs their hex equivalent?

Reply:
That change was based on input from @cabo.

Reply:
Could you link to the input?  Why have both?

Reply:
@cabo's [guidance](https://mailarchive.ietf.org/arch/msg/jsonpath/jip4OCSgTPH1LZYmL1iMCgxFiMk/) was to use hex for ranges and otherwise use quoted characters except where case sensitivity is needed.